module/condition/migrations/Version20201119103017.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Migration;
+
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20201119103017 extends AbstractErgonodeMigration
+{
+
+    public function up(Schema $schema): void
+    {
+        $languages = $this->getDictionaryActive();
+
+        $this->addSql(sprintf(
+            'WITH conditions_list as (
+                SELECT id, (\'{\'|| arr.position -1 || \',language}\')::text[] as path
```

Comment:
Is there any specific reason for not extracting just position and creating this in `jsonb_set` function? I'd rather have it like that